File: app/src/main/java/net/fionix/minima/WidgetService.java
package net.fionix.minima;

import android.content.Intent;
import android.widget.RemoteViewsService;

/**
 * Created by nazebzurati on 22/09/2017.
 */

public class WidgetService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return (new WidgetFactory(this.getApplicationContext(), intent));
    }
}

File: app/src/main/java/net/fionix/minima/widget/WidgetService.kt
package net.fionix.minima.widget

import android.content.Intent
import android.widget.RemoteViewsService

class WidgetService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return WidgetFactory(this.applicationContext, intent)
    }
}